components/base/fake/fake.go

```diff
@@ -93,6 +82,13 @@ func (b *Base) Close(ctx context.Context) error {
 	return nil
 }
 
+func (b *Base) Properties(ctx context.Context, extra map[string]interface{}) (map[base.Feature]float64, error) {
+	return map[base.Feature]float64{
+		base.TurningRadiusM: defaultMinimumTurningRadiusM,
+		base.WidthM:         defaultWidthMm * 0.001, // convert to meters
```

Comment:
[q] We don't want to just store the width as mm?

Reply:
The scope doc for this had meters in the API contract, so changed it here.

Reply:
Math for wheeled base and agileX has it in mm to match `MoveStraight` which expects a MM call, so left that there.